tests/lib/rules/no-multiple-empty-lines.js

```diff
@@ -286,6 +286,22 @@ ruleTester.run("no-multiple-empty-lines", rule, {
             output: "// invalid 21\n// unix line endings\nvar a = 5;\nvar b = 3;\n\n",
             errors: [ getExpectedErrorEOF(1) ],
             options: [ { max: 1 } ]
+        },
+        {
+            code:
+            "'foo';\n" +
+            "\n" +
+            "\n" +
+            "`bar`;\n" +
+            "`baz`;",
+            output:
+            "'foo';\n" +
+            "\n" +
+            "`bar`;\n" +
```

Comment:
I think there already are tests for that (specifically [these](/eslint/eslint/blob/ac3504d4f7542329bd271db369660099e2211e90/tests/lib/rules/no-multiple-empty-lines.js#L127-L147)). Am I misunderstanding the type of test you're referring to?